File: exercicios/fibonacci/fibonacci.js
/*
@author: Bruno Farina;
@description: Algoritmo Fibonacci;
@company: CEASB/PTI;
*/

fibonacci = function(n) {
	'use strict';
    var primeiro = 0,
        segundo = 1,
        prox, c;

    for (c = 0; c <= n; c++) {
        if (c <= 1) {
            prox = primeiro + segundo;
            console.log('[' + c + ']: ' + prox);
        } else {
            prox = primeiro + segundo;
            primeiro = segundo;
            segundo = prox;
            console.log('[' + c + ']: ' + prox);
        }
    }
};

File: exercicios/fibonacci/fibonacci.c
#include <stdio.h>

int main()
{
	int n = 10, primeiro = 0, segundo = 1, prox, c;
	
	for (c = 0; c <= n; c++){
		if (c <= 1){
			prox = c;
			printf("[%d]: %d \n", c, prox);
		}
		else {
			prox = primeiro + segundo;
			primeiro = segundo;
			segundo = prox;
			printf("[%d]: %d \n", c, prox);
		}
	}
	return 0;
}